strapi/src/api/member/content-types/member/schema.json

```
@@ -0,0 +1,52 @@
+{
+  "kind": "collectionType",
+  "collectionName": "members",
+  "info": {
+    "singularName": "member",
+    "pluralName": "members",
+    "displayName": "Member",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "link": {
+      "type": "string",
+      "required": true
+    },
+    "Role": {
+      "type": "string",
+      "required": true
+    },
+    "picture": {
+      "type": "media",
+      "multiple": false,
+      "required": false,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "name": {
+      "type": "string",
+      "required": true
+    },
+    "Affiliation": {
+      "type": "string",
+      "required": true
+    },
+    "pole": {
+      "type": "relation",
+      "relation": "manyToOne",
```

```suggestion
      "relation": "manyToMany",
```
I think that members could be in several poles.

---

Sounds weird but I agree it is not necessary to impose this in the database and wait for this exceptionnal situation to break the site